Editor/Resources/EditorWindow/Components/StatusBox.uxml

```diff
@@ -1,15 +1,13 @@
 ﻿<!-- Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved. -->
 <!-- SPDX-License-Identifier: Apache-2.0 -->
 
-<ui:UXML xmlns:ui="UnityEngine.UIElements">
+<ui:UXML xmlns:ui="UnityEngine.UIElements" class="status-box separator separator--horizontal">
     <ui:Style src="./statusbox.uss"/>
```

Comment:
This seems a little weird. It looks like we're going to be giving this component some spacing when it shouldn't get it.

Additionally, if I use this component in a file that hasn't imported `common.uss`, does that mean this won't be styled properly? If we include `<ui:Style src="./common.uss"/>` is the separator not going to work because it's outside the scope of the style import?